configuration_test.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import "testing"
+
+func TestConfigurationReadFileInvalid(t *testing.T) {
+	rc := func(n string) {
+		c := &configuration{}
+		if err := c.readFile(n); err == nil {
+			t.Error("expected error")
+		}
+	}
+
+	rc("")
+	rc("test/invalidConfiguration.toml")
+}
```

Comment:
Would it make sense to add a positive test here (e.g. valid configuration) even though it is implicitly tested elsewhere?
That way, we could have an explicit test

Reply:
What would this test cover?

Reply:
Loading a valid configuration or better different configuration files covering all cases (if more than one is needed)

Reply:
So basically we're testing the TOML library.

Reply:
Yeah, that wouldn't be clever. Maybe just leave it as is for now